src/index.ts

```diff
@@ -180,6 +183,8 @@ class Plugin implements TransformNode {
       needsCache: this._needsCache,
       volatile: this._volatile,
       trackInputChanges: this._trackInputChanges,
+      // @ts-ignore
```

Comment:
Why ignore this? Can you leave an inline comment linking to an issue that explains what needs to happen in order to remove the ts-ignore?